client/src/App/theme.js

```diff
@@ -0,0 +1,53 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+
+const theme = createMuiTheme({
+  palette: {
+   type: 'dark',
```

Comment:
why did you added  ` type: 'dark'` when you added  `background-color: black;` in index.css

Reply:
type is only for material-UI components, Wile background-color in CSS file for the entire page